bitnami/jaeger/templates/migrate-job.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "common.names.fullname" . }}-migrate
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: query
```

Comment:
I see this outside the `query` folder, I think that the `component` label is not necessary here. You could add the `part-of` one

Reply:
Its actually not part of the query anymore (it was while it was a jaeger-query initContainer). I just put `migration` as the component there on [d6f9cf5](https://github.com/bitnami/charts/pull/13480/commits/d6f9cf55c60f76acf8b9039f5bda12c01b992f55)